scenarios/evaluate/evaluate_endpoints/evaluate_endpoints.ipynb

```diff
@@ -209,26 +209,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from app_target import ModelEndpoints\n",
+    "from endpoint_target import ModelEndpoints\n",
```

Comment:
Lets bring this function into a cell in the notebook.
Make sure you add instructions to enable/disable each endpoint based on what the user has deployed, so they can comment out the models they don't have access to